src/main/java/org/broadinstitute/consent/http/util/InstitutionUtil.java

```diff
@@ -4,18 +4,20 @@
 import com.google.gson.FieldAttributes;
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
+import java.time.Instant;
 import java.util.List;
 import java.util.Objects;
 import org.broadinstitute.consent.http.enumeration.UserRoles;
 import org.broadinstitute.consent.http.models.User;
 import org.broadinstitute.consent.http.models.UserRole;
+import org.broadinstitute.consent.http.util.gson.InstantTypeAdapter;
 
 public class InstitutionUtil implements ConsentLogger {
 
   private final GsonBuilder gson;
 
   public InstitutionUtil() {
-    this.gson = new GsonBuilder();
+    this.gson = new GsonBuilder().registerTypeAdapter(Instant.class, new InstantTypeAdapter());
```

Comment:
I recommend you add this adapter to `GsonUtil.gsonBuilderWithAdapters` in `GsonUtil`, and then call that here.